src/rgw/motr/synchronization/rgw_motr_gc_sync.cc

```diff
@@ -0,0 +1,77 @@
+// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
+// vim: ts=2 sw=2 expandtab ft=cpp
+
+/*
+ * Ceph - scalable distributed file system
+ *
+ * SAL implementation for the CORTX Motr backend
+ *
+ * Copyright (C) 2021 Seagate Technology LLC and/or its Affiliates
+ *
+ * This is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License version 2.1, as published by the Free Software
+ * Foundation. See file COPYING.
+ *
+ */
+
+#include "motr/synchronization/rgw_motr_gc_sync.h"
```

Comment:
I think we can shorten the file names, `motr/sync/gc_sync.h`, also if these files are related to gc then should we put them under `motr/gc` dir?

Reply:
I agree, This is also independent of GC, so gc shouldn't be mentioned in the code or file name.

Reply:
doable